sdk/mobile-center-distribute/src/main/java/com/microsoft/azure/mobile/distribute/Distribute.java

```diff
@@ -641,21 +649,28 @@ else if (mUnknownSourcesDialog != null) {
                 return;
             }
 
-            /* Check if we have previous stored the update token. */
+            /*
+             * Check if we have previously stored the redirection parameters,
+             * note that distribution group is missing in previous SDK releases.
```

Comment:
Previous as in 0.11.0.
In the future the or will be replaced by an and.
Again writing code that way makes it more simpler and is future proof.